runtime/oti/j9nonbuilder.h

```diff
@@ -2048,7 +2048,7 @@ typedef struct J9BCTranslationData {
 #define BCT_MajorClassFileVersionMaskShift  24
 
 /* A given Java feature version uses class-file format (version) + 44. */
-#define BCT_JavaMajorVersionShifted(java_version)  ((44 + (I_32)(java_version)) << BCT_MajorClassFileVersionMaskShift)
+#define BCT_JavaMajorVersionShifted(java_version)  ((44 + (U_32)(java_version)) << BCT_MajorClassFileVersionMaskShift)
```

Comment:
You could cast the whole thing, does that help?
`((U_32)((44 + (U_32)(java_version)) << BCT_MajorClassFileVersionMaskShift))`

Reply:
No, tried and didn't help.